### ISSUE TITLE
How to vizualize graphic ML vs L

### ISSUE THREAD

**Issue by litvinen** [StochasticNumerics/mimclib#108]
Hello,
I would like to see convergence/decay on graphic ML(number of samples on each level) vs level L.
Is it possible?
Thanks

**Comment by litvinen** [comment]
May be to use function 
def plotWorkVsLvlStats(ax, runs, *args, **kwargs): 
from plot.py ?

**Comment by haji-ali** [comment]
I don't think we ever had this plot. You can write a plotting function to that though.
